build/gulpfile.editor.js

```diff
@@ -40,7 +40,6 @@ const editorEntryPoints = [
 		exclude: ['vs/nls'],
 		prepend: [
 			{ path: 'vs/loader.js' },
-			{ path: 'vs/nls.js', amdModuleId: 'vs/nls' },
```

Comment:
Indeed, I think this needs to stay. Even after adding this line back, when I run locally `yarn gulp editor-distro`, it now fails with the following:
```
[09:27:49] The following tasks did not complete: editor-distro
[09:27:49] Did you forget to signal async completion?
error Command failed with exit code 1.
```
This almost looks like we forget to return a Promise or a stream somewhere

Reply:
Maybe fixed with https://github.com/microsoft/vscode/pull/160957/commits/f5e095bc4d74b6969d55a55b35487ff048f84be5?